src/test/java/com/github/onsdigital/babbage/api/endpoint/util/LegacyProxyMaxAgeTest.java

```diff
@@ -0,0 +1,78 @@
+package com.github.onsdigital.babbage.api.endpoint.util;
+
+import static org.junit.Assert.assertEquals;
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.mockStatic;
+import static org.mockito.Mockito.when;
+
+import java.util.Arrays;
+import java.util.Collections;
+
+import javax.servlet.http.HttpServletRequest;
+
+import org.apache.http.client.methods.CloseableHttpResponse;
+import org.apache.http.client.methods.HttpUriRequest;
+import org.apache.http.client.methods.RequestBuilder;
+import org.apache.http.impl.client.CloseableHttpClient;
+import org.apache.http.impl.client.HttpClients;
+import org.apache.http.message.BasicHeader;
+import org.junit.Before;
+import org.junit.Test;
+import org.mockito.MockedStatic;
+
+import com.github.onsdigital.babbage.api.util.LegacyProxyMaxAge;
+
+public class LegacyProxyMaxAgeTest {
+    private CloseableHttpClient mockHttpClient;
+    private CloseableHttpResponse mockResponse;
+    private HttpServletRequest mockRequest;
+    private RequestBuilder mockRequestBuilder;
+    private HttpUriRequest mockHttpUriRequest;
+
+    @Before
+    public void setup() throws Exception {
+        mockResponse = mock(CloseableHttpResponse.class);
+        mockRequestBuilder = mock(RequestBuilder.class);
+        mockHttpUriRequest = mock(HttpUriRequest.class);
+        when(mockRequestBuilder.build()).thenReturn(mockHttpUriRequest);
+
+        mockRequest = mock(HttpServletRequest.class);
+        when(mockRequest.getMethod()).thenReturn("GET");
+        when(mockRequest.getHeaderNames()).thenReturn(Collections.enumeration(Arrays.asList("Accept-Language", "User-Agent")));
+        when(mockRequest.getHeader("Accept-Language")).thenReturn("en-US");
+        when(mockRequest.getHeader("User-Agent")).thenReturn("Mozilla/5.0");
+    }
+
+    @Test
+    public void testGetMaxAgeValueFromLegacyProxy_WhenCacheControlHeaderIsNull() throws Exception {
+        try (MockedStatic<HttpClients> mockedHttpClients = mockStatic(HttpClients.class)) {
```

Comment:
You seem to need `mockedHttpClients` in every test, so instead of declaring this try-with-resources, you could set this up in the `setup` method, and then create a new `tearDown` method (with the `@After` annotation), in which you call `mockedHttpClients.close()`.

Reply:
I initially declared this in the setup method but for some reason the mocked behaviour of the httpclient is not reset before each test, causing the second test to fail, so I moved the HttpClients.createDefault into each test method to ensure it resets for each test

Reply:
Does that still happens if you call `close()` in the `@After` method?

Reply:
Yes, I tried that as well and it wouldn't reset for the second test

Reply:
I think you may have done something wrong, because it worked for me. Have a look at my changes: 487f1ba8fdcc6c31e1595a2e78e39acf6dd0a7ba